app.js

```diff
@@ -0,0 +1,39 @@
+var createError = require('http-errors');
```

Comment:
Use `const` instead of `var` for all the variables that never change value.
For example: Imported modules